### ISSUE TITLE
JSON files should be formatted on rendering

### ISSUE THREAD

**Issue by vlaaad** [defold/editor2-issues#2414]
Opening unformatted JSON files (such as ones exported from spine) are displayed as one long line, making them difficult to read. It would be nice to have a default viewer for json files that at least can format them.

**Comment by AGulev** [comment]
I don't think so, I think it's better to have "as is" because I wanna see how it looks like, and minify it if need.

**Comment by vlaaad** [comment]
You still will be able to see them minified if you open them as text. I think keeping json files minified is a good idea overall, because it saves you extra space wasted on formatting, and ability to view them in formatted form makes keeping them more convenient.
